### PR TITLE
S3 access logs expiration

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/2726

### CODE REVIEW

**Comment by pipo02mix** [comment]
I have tested in ginger and it worked. PTAL

File: service/controller/v10/resource/s3bucket/resource.go [hunk 1]
```diff
@@ -60,6 +64,9 @@ func New(config Config) (*Resource, error) {
 	}
 
 	// Settings.
+	if config.AccessLogsExpiration < 0 {
+		return nil, microerror.Maskf(invalidConfigError, "%T.AccessLogsExpiration must not be lower than 0", config)
+	}
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply service/controller/v10/resource/s3bucket/resource.go]: It is in the default configmap and has a default value in the flag. In the sync I talked about the removal of the logging buckets in the test envs(this is for setting the expiration if the logs entries). I will do next week a PR to include the configuration for the test so we can react and delete logging bucket when cluster is deleted

**Comment by pipo02mix** [comment]
BTW Tim, I could not make pass the test `e2eTestWIPPRDraining` before. There were different issues as I commented on slack. I will give it another try

**Comment by xh3b4sd** [comment]
What kind of issues are there? Is it always only the draining test? Do we know why it fails? We have to fix this shit. This is really annoying. I feel you.

**Comment by pipo02mix** [comment]
Now all tests passes :) Merging!

**Pull request merged**